docs/rag.md

```diff
@@ -78,21 +81,63 @@ with a stable source of data. On the other hand, if you are working with data
 that frequently changes, you might continuously run the ingestion flow (for
 example, in a Cloud Firestore trigger, whenever a document is updated).
 
-The following example shows how you could ingest a collection of PDF documents
-into a vector database. It uses the local file-based vector similarity retriever
+### Embedders
+
+An embedder is a function that takes a collection of content and creates a numeric vector that is representative of the semantic meaning of the original text. As mentioned above, embedders are leveraged as part of the process of indexing, however, they can also be used independently to create embeddings independent of an index.
+
+### Retrievers
+
+A retriever is a concept that encapsulates logic related to any kind of document
+retrieval. The most popular retrieval cases typically include retrieval from
```

Comment:
nit; recommend changing `typically` to often in this line to avoid over using the word

Reply:
I'm noticing I say independently too much as well - will fix!

Reply:
done